{
  "document": {
    "aggregate_severity": {
      "namespace": "https://www.suse.com/support/security/rating/",
      "text": "moderate"
    },
    "category": "csaf_vex",
    "csaf_version": "2.0",
    "distribution": {
      "text": "Copyright 2023 SUSE LLC. All rights reserved.",
      "tlp": {
        "label": "WHITE",
        "url": "https://www.first.org/tlp/"
      }
    },
    "lang": "en",
    "notes": [
      {
        "category": "summary",
        "text": "SUSE CVE-2013-2132",
        "title": "Title"
      },
      {
        "category": "description",
        "text": "bson/_cbsonmodule.c in the mongo-python-driver (aka. pymongo) before 2.5.2, as used in MongoDB, allows context-dependent attackers to cause a denial of service (NULL pointer dereference and crash) via vectors related to decoding of an \"invalid DBRef.\"",
        "title": "Description of the CVE"
      },
      {
        "category": "legal_disclaimer",
        "text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
        "title": "Terms of use"
      }
    ],
    "publisher": {
      "category": "vendor",
      "contact_details": "https://www.suse.com/support/security/contact/",
      "name": "SUSE Product Security Team",
      "namespace": "https://www.suse.com/"
    },
    "references": [
      {
        "category": "external",
        "summary": "CVE-2013-2132",
        "url": "https://www.suse.com/security/cve/CVE-2013-2132"
      },
      {
        "category": "external",
        "summary": "SUSE Security Ratings",
        "url": "https://www.suse.com/support/security/rating/"
      },
      {
        "category": "external",
        "summary": "SUSE Bug 822798 for CVE-2013-2132",
        "url": "https://bugzilla.suse.com/822798"
      },
      {
        "category": "external",
        "summary": "Advisory link for openSUSE-SU-2013:1064-1",
        "url": "https://lists.opensuse.org/opensuse-updates/2013-06/msg00180.html"
      }
    ],
    "title": "SUSE CVE CVE-2013-2132",
    "tracking": {
      "current_release_date": "2023-02-15T05:39:06Z",
      "generator": {
        "date": "2023-02-15T05:39:06Z",
        "engine": {
          "name": "cve-database.git:bin/generate-csaf-vex.pl",
          "version": "1"
        }
      },
      "id": "CVE-2013-2132",
      "initial_release_date": "2023-02-15T05:39:06Z",
      "revision_history": [
        {
          "date": "2023-02-15T05:39:06Z",
          "number": "2",
          "summary": "Current version"
        }
      ],
      "status": "interim",
      "version": "2"
    }
  },
  "product_tree": {
    "branches": [
      {
        "branches": [
          {
            "branches": [
              {
                "category": "product_name",
                "name": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
                "product": {
                  "name": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
                  "product_id": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
                  "product_identification_helper": {
                    "cpe": "cpe:/o:suse:suse-sle12-cloud-compute:5"
                  }
                }
              },
              {
                "category": "product_name",
                "name": "SUSE OpenStack Cloud 6",
                "product": {
                  "name": "SUSE OpenStack Cloud 6",
                  "product_id": "SUSE OpenStack Cloud 6",
                  "product_identification_helper": {
                    "cpe": "cpe:/o:suse:suse-openstack-cloud:6"
                  }
                }
              },
              {
                "category": "product_name",
                "name": "openSUSE Tumbleweed",
                "product": {
                  "name": "openSUSE Tumbleweed",
                  "product_id": "openSUSE Tumbleweed",
                  "product_identification_helper": {
                    "cpe": "cpe:/o:opensuse:tumbleweed"
                  }
                }
              },
              {
                "category": "product_version",
                "name": "python-pymongo-2.6.3-2.20",
                "product": {
                  "name": "python-pymongo-2.6.3-2.20",
                  "product_id": "python-pymongo-2.6.3-2.20"
                }
              },
              {
                "category": "product_version",
                "name": "python-pymongo-3.0.3-1.1",
                "product": {
                  "name": "python-pymongo-3.0.3-1.1",
                  "product_id": "python-pymongo-3.0.3-1.1"
                }
              },
              {
                "category": "product_version",
                "name": "python-pymongo-3.1.1-1.5",
                "product": {
                  "name": "python-pymongo-3.1.1-1.5",
                  "product_id": "python-pymongo-3.1.1-1.5"
                }
              },
              {
                "category": "product_version",
                "name": "python3-pymongo-3.4.0-1.1",
                "product": {
                  "name": "python3-pymongo-3.4.0-1.1",
                  "product_id": "python3-pymongo-3.4.0-1.1"
                }
              },
              {
                "category": "product_version",
                "name": "python36-pymongo-3.11.4-1.2",
                "product": {
                  "name": "python36-pymongo-3.11.4-1.2",
                  "product_id": "python36-pymongo-3.11.4-1.2"
                }
              },
              {
                "category": "product_version",
                "name": "python38-pymongo-3.11.4-1.2",
                "product": {
                  "name": "python38-pymongo-3.11.4-1.2",
                  "product_id": "python38-pymongo-3.11.4-1.2"
                }
              },
              {
                "category": "product_version",
                "name": "python39-pymongo-3.11.4-1.2",
                "product": {
                  "name": "python39-pymongo-3.11.4-1.2",
                  "product_id": "python39-pymongo-3.11.4-1.2"
                }
              }
            ],
            "category": "product_family",
            "name": "SUSE Linux Enterprise"
          }
        ],
        "category": "vendor",
        "name": "SUSE"
      }
    ],
    "relationships": [
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python-pymongo-2.6.3-2.20 as component of SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5",
          "product_id": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5:python-pymongo-2.6.3-2.20"
        },
        "product_reference": "python-pymongo-2.6.3-2.20",
        "relates_to_product_reference": "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python-pymongo-3.0.3-1.1 as component of SUSE OpenStack Cloud 6",
          "product_id": "SUSE OpenStack Cloud 6:python-pymongo-3.0.3-1.1"
        },
        "product_reference": "python-pymongo-3.0.3-1.1",
        "relates_to_product_reference": "SUSE OpenStack Cloud 6"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python-pymongo-3.1.1-1.5 as component of openSUSE Tumbleweed",
          "product_id": "openSUSE Tumbleweed:python-pymongo-3.1.1-1.5"
        },
        "product_reference": "python-pymongo-3.1.1-1.5",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python3-pymongo-3.4.0-1.1 as component of openSUSE Tumbleweed",
          "product_id": "openSUSE Tumbleweed:python3-pymongo-3.4.0-1.1"
        },
        "product_reference": "python3-pymongo-3.4.0-1.1",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python36-pymongo-3.11.4-1.2 as component of openSUSE Tumbleweed",
          "product_id": "openSUSE Tumbleweed:python36-pymongo-3.11.4-1.2"
        },
        "product_reference": "python36-pymongo-3.11.4-1.2",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python38-pymongo-3.11.4-1.2 as component of openSUSE Tumbleweed",
          "product_id": "openSUSE Tumbleweed:python38-pymongo-3.11.4-1.2"
        },
        "product_reference": "python38-pymongo-3.11.4-1.2",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      },
      {
        "category": "default_component_of",
        "full_product_name": {
          "name": "python39-pymongo-3.11.4-1.2 as component of openSUSE Tumbleweed",
          "product_id": "openSUSE Tumbleweed:python39-pymongo-3.11.4-1.2"
        },
        "product_reference": "python39-pymongo-3.11.4-1.2",
        "relates_to_product_reference": "openSUSE Tumbleweed"
      }
    ]
  },
  "vulnerabilities": [
    {
      "cve": "CVE-2013-2132",
      "ids": [
        {
          "system_name": "SUSE CVE Page",
          "text": "https://www.suse.com/security/cve/CVE-2013-2132"
        }
      ],
      "notes": [
        {
          "category": "general",
          "text": "bson/_cbsonmodule.c in the mongo-python-driver (aka. pymongo) before 2.5.2, as used in MongoDB, allows context-dependent attackers to cause a denial of service (NULL pointer dereference and crash) via vectors related to decoding of an \"invalid DBRef.\"",
          "title": "Vulnerability description"
        }
      ],
      "product_status": {
        "fixed": [
          "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5:python-pymongo-2.6.3-2.20",
          "SUSE OpenStack Cloud 6:python-pymongo-3.0.3-1.1",
          "openSUSE Tumbleweed:python-pymongo-3.1.1-1.5",
          "openSUSE Tumbleweed:python3-pymongo-3.4.0-1.1",
          "openSUSE Tumbleweed:python36-pymongo-3.11.4-1.2",
          "openSUSE Tumbleweed:python38-pymongo-3.11.4-1.2",
          "openSUSE Tumbleweed:python39-pymongo-3.11.4-1.2"
        ]
      },
      "references": [
        {
          "category": "external",
          "summary": "CVE-2013-2132",
          "url": "https://www.suse.com/security/cve/CVE-2013-2132"
        },
        {
          "category": "external",
          "summary": "SUSE Security Ratings",
          "url": "https://www.suse.com/support/security/rating/"
        },
        {
          "category": "external",
          "summary": "SUSE Bug 822798 for CVE-2013-2132",
          "url": "https://bugzilla.suse.com/822798"
        },
        {
          "category": "external",
          "summary": "Advisory link for openSUSE-SU-2013:1064-1",
          "url": "https://lists.opensuse.org/opensuse-updates/2013-06/msg00180.html"
        }
      ],
      "remediations": [
        {
          "category": "vendor_fix",
          "details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
          "product_ids": [
            "SUSE Cloud Compute Node for SUSE Linux Enterprise 12 5:python-pymongo-2.6.3-2.20",
            "SUSE OpenStack Cloud 6:python-pymongo-3.0.3-1.1",
            "openSUSE Tumbleweed:python-pymongo-3.1.1-1.5",
            "openSUSE Tumbleweed:python3-pymongo-3.4.0-1.1",
            "openSUSE Tumbleweed:python36-pymongo-3.11.4-1.2",
            "openSUSE Tumbleweed:python38-pymongo-3.11.4-1.2",
            "openSUSE Tumbleweed:python39-pymongo-3.11.4-1.2"
          ]
        }
      ],
      "threats": [
        {
          "category": "impact",
          "date": "2013-08-09T14:01:10Z",
          "details": "moderate"
        }
      ],
      "title": "CVE-2013-2132"
    }
  ]
}
